build.gradle.kts

```diff
@@ -221,7 +221,7 @@ tasks.register<GenerateTask>("buildPrisonApiModel") {
   )
   globalProperties.set(
     mapOf(
-      "models" to "",
+      "models" to "PrisonerInPrisonSummary,PrisonPeriod,SignificantMovement,TransferDetail",
```

Comment:
Not related to prisoner-search-api, but I decided to modify the codegen for prison-api so that it only generates the models that we are actually using (prison-api is massive and it was generating loads of models - I'm hoping this change might make a (small) improvement in build times? 🤞 )